dev/apollo-federation/supergraph.graphql

```diff
@@ -0,0 +1,1828 @@
+schema
```

Comment:
some README or pointer would be useful. I'm not too sure how to interpret this file right now without more context

Reply:
it gets generated in the `yarn write-sdl` task. Will think about where to document it.

Reply:
Added a readme under `dev/apollo-federation`